libraries/bxdf/standard_surface.mtlx

```diff
@@ -47,12 +53,24 @@
     <input name="normal" type="vector3" defaultgeomprop="Nworld" uiname="Normal" uifolder="Geometry" doc="Input geometric normal" />
     <input name="tangent" type="vector3" defaultgeomprop="Tworld" uiname="Tangent Input" uifolder="Geometry" doc="Input geometric tangent" />
     <output name="out" type="surfaceshader" />
-  </nodedef>
+   </nodedef> 
+   
+   <!--
+      Association between imlementation and definition.
+      Note that version 1.0.1 only changes some default values and thus reuses the same 1.0.0 nodegraph implementation
+   -->
+   <implementation name="IMPL_standard_surface_surfaceshader_101"
```

Comment:
We can now correctly association 1 nodegraph impl with 2 nodedefs which have different default values.
For backwards compatibility the current nodedef name was not changed.